followthemoney/schema/CallForTenders.yaml

```diff
@@ -0,0 +1,77 @@
+CallForTenders:
```

Comment:
Would `Tender` also work? I'm just trying to keep the names simple, but this may be less precise.

Reply:
I do understand but I don't think so; this is the process of calling companies (tenderers) to make an offer / a tender. And then, from the call, one contract (or more, depending on number of lots I think) is awarded to one or more tenderer(s).